<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommunityFundCreateProposalDialog</class>
 <widget class="QDialog" name="CommunityFundCreateProposalDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Proposal</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutProposalDialog">
     <item>
      <widget class="QLabel" name="labelCreateProposalTitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Create Proposal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelNavcoinAddressTitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Navcoin Address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QValidatedLineEdit" name="lineEditNavcoinAddress"/>
     </item>
     <item>
      <widget class="QLabel" name="labelRequestedAmountTitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Requested Amount (NAV):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="NavcoinAmountField" name="lineEditRequestedAmount"/>
     </item>
     <item>
      <widget class="QLabel" name="labelDescriptionTitle">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Title + URL:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="plainTextEditDescription"/>
     </item>
     <item>
      <widget class="QLabel" name="labelProposalDurationTitle">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Proposal Duration:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutDuration">
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutDays">
         <item>
          <widget class="QValidatedSpinBox" name="spinBoxDays">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelDaysTitle">
           <property name="text">
            <string>Days</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutHours">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QValidatedSpinBox" name="spinBoxHours"/>
         </item>
         <item>
          <widget class="QLabel" name="labelHoursTitle">
           <property name="text">
            <string>Hours</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutMinutes">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QValidatedSpinBox" name="spinBoxMinutes"/>
         </item>
         <item>
          <widget class="QLabel" name="labelMinutesTitle">
           <property name="text">
            <string>Minutes</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="superProposalCheckbox">
       <property name="text">
        <string>Super proposal (request to print coins)</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutCreateProposalButton">
       <item>
        <widget class="QPushButton" name="pushButtonCreateProposal">
         <property name="text">
          <string>Create Proposal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelWarning">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>By submitting the proposal a X NAV deduction will occur from your wallet </string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavcoinAmountField</class>
   <extends>QLineEdit</extends>
   <header>navcoinamountfield.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qvalidatedlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>QValidatedSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qvalidatedspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
